.github/workflows/cherry-pick-image.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
           GOOGLE_APPLICATION_CREDENTIALS: /tmp/gcr-sa.json
         run: |
           # write token
-          base64 --decode <<< ${{ secrets.GCR_SA_B64_CREDENTIALS }} | jq > ${GOOGLE_APPLICATION_CREDENTIALS}
+          base64 --decode <<< ${{ secrets.GCR_SA_B64_CREDENTIALS }} > ${GOOGLE_APPLICATION_CREDENTIALS}
```

Comment:
This change is fallout from a slack conversation where it was determined that the original error was not due to formatting, but instead was due to a missing `GCR_SA_B64_CREDENTIALS` secret in jade-data-repo-ui. There are similar changes in `staging-smoke-tests.yaml`